class.jetpack.php

```diff
@@ -560,6 +564,19 @@ private function __construct() {
 		}
 	}
 
+	function rest_api_init() {
+		register_rest_route( 'jetpack/v1', '/verify_registration', array(
+			'methods' => 'POST',
```

Comment:
This should probably use the `WP_REST_Server::EDITABLE` constant instead of manually writing out `POST`